apc:
  tf_origin_to_odom:
    x: 1.32
    y: 0.6
    z: 0.0
  tf_shelf_to_origin:
    x: 0.0
    y: 0.0
    z: 0.86
